samples/cpp/speech_sample/README.md

```diff
@@ -99,24 +99,22 @@ speech_sample [OPTION]
 Options:
 
     -h                         Print a usage message.
-    -i "<path>"                Required. Paths to input files. Example of usage: <file1.ark,file2.ark> or <file.ark> or <file.npz>.
+    -i "<path>"                Required. Paths to input file or Layers names with corresponding paths to the input files. Example of usage for single file: <file.ark> or <file.npz>. Example of usage for named layers: <layer1>=<file1.ark>,<layer2>=<file2.ark>.
```

Comment:
Is there an option for multiple files right now (without naming them of course)? Example from previous state of the sample: `<file1.ark,file2.ark>`

Reply:
Here we just updated documentation and helper to current implementation. It was done few months ago.

Previously for these purpose you had to use -i and -iname together.
Now only -i is needed.

Reply:
I see. Probably in future there will be a request to support any input list but for now it is ok.

Reply:
You can use any input list right now, but you need to provide it in pairs input_name=file_name. Otherwise it was ambiguous which input would be used for a certain file. The only case when file_name was enough was a single input model case.

Reply:
It is not so ambiguous for the potential users. At least Python API provides such option to pass inputs as a list without any indication on names. Python API handles it in a way that enumeration of inputs (Tensors/numpy arrays to be precise)  is done automatically in the and uses `set_input_tensor` in a loop. It is just a heads up for potential upgrades to the sample (or even core C++ API).

Reference to exact `set_input_tensor` overload: https://github.com/openvinotoolkit/openvino/blob/bb82f43e22486efefced7483fa5d1cc4be7b1f0f/src/inference/include/openvino/runtime/infer_request.hpp#L159

Reply:
Sure :-)